xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5051,7 +5051,7 @@ xe.blog.code.notblog=This is not a blog page!
 xe.blog.code.published=This blog post is not published yet.
 xe.blog.code.hidden=This blog post is hidden.
 xe.blog.code.notpublished=This blog post is not published yet. Publish it.
-xe.blog.code.madevisible=Made entry visible
+xe.blog.code.madevisible=Entry is made visible.
```

Comment:
Are you sure about this one? The new phrasing looks incorrect. I'd venture that the correct way would be to say "Entry has been made visible" rather than "Entry is made visible" which seems more incorrect than the previous translation... :(

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/commit/5b55d576455ae6305e7f8e9b4b98c90f2350c161